### ISSUE TITLE
Current status of model training

### ISSUE THREAD

**Issue by kuke** [PaddlePaddle/models#737]
We are training the DeepASR model on the whole training dataset with duration 2000h. After 12 epoches' training, the model has well converged already.  

Settings:
```
batch_size: 128
device: GPU
hidden_dim: 1024
learning_rate: 0.00016
minimum_batch_size: 1
proj_dim: 512
stacked_num: 5
optimizer: Adam
```

Env: 4 P40 GPUs, 15h per epoch.

![training_acc_on_all_data](https://user-images.githubusercontent.com/3064195/37443696-9dc6fb18-2848-11e8-9af7-81a3b2cff568.png)

After the decoder is ready, we will continue to fine tune this model to catch up with the performance in accuracy of benchmark.

**Comment by shanyi15** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持! 
 Hello, this issue has not been updated in the past month. We will close it today for the sake of other user‘s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!
